pkg/daemon/ceph/client/mirror.go

```diff
@@ -17,21 +17,41 @@ limitations under the License.
 package client
 
 import (
+	"encoding/base64"
 	"encoding/json"
 	"fmt"
 	"io/ioutil"
 	"os"
+	"strings"
 
 	"github.com/pkg/errors"
 	cephv1 "github.com/rook/rook/pkg/apis/ceph.rook.io/v1"
 	"github.com/rook/rook/pkg/clusterd"
+	"github.com/rook/rook/pkg/util"
+)
+
+// PeerToken is the content of the peer token
+type PeerToken struct {
+	ClusterFSID string `json:"fsid"`
+	ClientID    string `json:"client_id"`
+	Key         string `json:"key"`
+	MonHost     string `json:"mon_host"`
+	// These fields are added by Rook and NOT part of the output of client.CreateRBDMirrorBootstrapPeer()
+	PoolID    int    `json:"pool_id"`
```

Comment:
@leseb can we remove pool_id? this is not much helpful for 3 reason
* As we are going to have mapping of pool Id's
* As this token is per ceph cluster we will have multiple pool_id's in a cluster?
* And when this token is generated the pool ID will always be empty?

Reply:
I'll remove that in a follow-up PR.